pkg/reconciler/common/transformers.go

```diff
@@ -344,6 +364,28 @@ func TaskImages(images map[string]string) mf.Transformer {
 	}
 }
 
+func getStepActionName(stepActionData interface{}) string {
+	name, ok := stepActionData.(string)
+	if ok && name != "" {
+		return name
+	}
+	log.Println("Unable to get the stepaction name", name)
```

Comment:
@savitaashture use `*zap.SugaredLogger` from the context

Reply:
done